src/components/pages/GamePlay/GameMissionProgress.js

```diff
@@ -31,37 +31,99 @@ export default function GameMissionProgress(props) {
     }
   };
 
+  // const showModal = () => {
+  //   setIsModalVisible(true);
+  //   const modalData = {
+  //     title: 'Quick Navigation Tips!',
+  //     description:
+  //       'Click on the "Read," "Draw" or "Write" buttons to navigate through the mission. You have to complete your current step before you can move on to the next step, so if you haven\'t completed the "Read" step, you won\'t be able to click on "Draw" or "Write" to advance. Once you have completed all the steps, you can click back and forth between all of them.',
+  //     buttonTxt: 'Got it!',
+  //   };
+
+  //   props.enableModalWindow(modalData);
+  // };
+
+  const showGameTips = () => {
+    return (
+      <div className="game-tips-container">
+        <div className="game-tips-text">
+          <p className="game-tips-text-p">
+            <img
+              style={{ width: '48px', height: '48px' }}
+              src="https://img.icons8.com/flat-round/64/000000/wide-right-arrow.png"
+            />{' '}
+            Click on the "Read," "Draw" or "Write" buttons to navigate through
+            the mission.
+          </p>
+          <p className="game-tips-text-p">
+            <img
+              style={{ width: '48px', height: '48px' }}
+              src="https://img.icons8.com/flat-round/64/000000/wide-right-arrow.png"
+            />{' '}
+            You have to complete your current step before you can move on to the
+            next step
+          </p>
+          <p className="game-tips-text-p">
+            <img
+              style={{ width: '48px', height: '48px' }}
+              src="https://img.icons8.com/flat-round/64/000000/wide-right-arrow.png"
+            />{' '}
+            Once you have completed all the steps, you can click back and forth
+            between all of them.
+          </p>
+        </div>
+        <button className="game-tips-gotit-btn">Got it!</button>
```

Comment:
Should use the ant design Button instead. Then the cursor will change to a pointing hand on hover

Reply:
> Should use the ant design Button instead. Then the cursor will change to a pointing hand on hover

Thank you, Chris. I just changed the cursor into a Hand Pointer when the user hovers on the button as you suggested. It does make the button harmonious with the other button in the entire app. However, the design team agreed that the "Got it" button should have the same color as the "read, draw, write" container on the step page, and I need to design the size, font size, and font family of the "Got it" button to harmonize with other elements in the info-box, if I use the ```<Button/>``` component, I need to add ```!important``` into every line of my style code, that's why I chose to create ```<button/> tag instead.
![gotitbtn](https://user-images.githubusercontent.com/72033605/147834365-81c5d4a5-ba8e-452d-a980-932d9586eb89.png)
![gotitbtn2](https://user-images.githubusercontent.com/72033605/147834375-164817c2-3698-4fc2-8c62-7be2fb874de3.png)